readme.rst

```diff
@@ -1,6 +1,20 @@
 Coding Guideline for TYPO3
 ==========================
 
+Installation
+------------
+
+You have to require the package via composer `composer require codappix/cgl-typo3`.
+After that you have to add the following to the `composer.json`:
```

Comment:
Shouldn't this be added before? Otherwise we should point out that you have to run `composer dumpautoload` afterwards.

Reply:
Shouldn't we use a code-block? Currently the `composer.json` part is rendered as quote, right?